search/search-client/src/main/java/io/camunda/search/clients/CamundaSearchClients.java

```diff
@@ -322,7 +322,7 @@ public SearchQueryResult<GroupMemberEntity> searchGroupMembers(final GroupQuery
   @Override
   public UserEntity getUser(final String id) {
```

Comment:
🔧 The parameter could be also changed to username to make it more consistent:

```suggestion
  public UserEntity getUser(final String username) {
```